src/bot/api.rs

```diff
@@ -1707,19 +1707,921 @@ impl Bot {
 impl Requester for Bot {
     type Err = crate::errors::RequestError;
 
+    type GetUpdates = JsonRequest<payloads::GetUpdates>;
+
+    fn get_updates(&self) -> Self::GetUpdates where {
+        Self::GetUpdates::new(self.clone(), payloads::GetUpdates::new())
+    }
+
+    type SetWebhook = JsonRequest<payloads::SetWebhook>;
+
+    fn set_webhook<U, A>(&self, url: U, allowed_updates: A) -> Self::SetWebhook
+    where
+        U: Into<String>,
+        A: IntoIterator<Item = crate::types::AllowedUpdate>,
+    {
+        Self::SetWebhook::new(self.clone(), payloads::SetWebhook::new(url, allowed_updates))
+    }
+
+    type DeleteWebhook = JsonRequest<payloads::DeleteWebhook>;
+
+    fn delete_webhook(&self) -> Self::DeleteWebhook where {
+        Self::DeleteWebhook::new(self.clone(), payloads::DeleteWebhook::new())
+    }
+
+    type GetWebhookInfo = JsonRequest<payloads::GetWebhookInfo>;
+
+    fn get_webhook_info(&self) -> Self::GetWebhookInfo where {
+        Self::GetWebhookInfo::new(self.clone(), payloads::GetWebhookInfo::new())
+    }
```

Comment:
Empty `where` blocks.
May be also somewhere else.

Reply:
Huh, I thought rustfmt handles this...